docs/tools/CLI/cli-compile.md

```diff
@@ -35,38 +53,33 @@ Total RAM memory (data + bss + heap + stack): 109096 bytes
 Total Flash memory (text + data + misc): 66014 bytes
 Image: BUILD/K64F/GCC_ARM/mbed-os-program.bin
 ```
-### Set the target
 
-You can set the default target with the `mbed target` command to avoid having to supply the `-m` argument:
+### Set default or detected target
```

Comment:
"Set ... detected target" sounds a bit off to me. You can't "set" something that is "detected". I personally prefer the original (or something like "Set default target").